packages/flutter/test/material/data_table_theme_test.dart

```diff
@@ -60,6 +64,8 @@ void main() {
         (Set<MaterialState> states) => const Color(0xfffffff1),
       ),
       dataRowHeight: 51.0,
+      dataRowMinHeight: 41.0,
+      dataRowMaxHeight: 42.0,
```

Comment:
```suggestion
      dataRowHeight: 51.0,
      dataRowMinHeight: 41.0,
      dataRowMaxHeight: 42.0,
```

These seem to conflict with each other. How can the height be 41<=x<=42, but also x=51? There should be an assert that let's the user know this is not allowed.